### DISCUSSION TITLE
Source question

### DISCUSSION THREAD

**Discussion by katastrophe92** [Devolutions/UniGetUI#1480] [Q&A]
![11 October 22023 ⁓ 07⁚50⁚30 ⁓ 958  ⁄926](https://github.com/marticliment/WingetUI/assets/103950507/e151e9ea-9da7-4d67-9d51-aa2e7e286296)
Hello WingetUI community! First of all @marticliment thank you for this amazing project, I can't think of installing software's the ''normal'' way after finding this project😂. 
What is the difference between this 3 sources: Winget, Winget: msstore and Winget: winget?
Are Scoop and Chocolatey apps installed as Portable apps?
Hope someone can help me. Thanks!

**Comment by marticliment** [comment]
Hello @katastrophe92,

Winget: winget apps are available from the source Winget from the Winget Package Manager
Winget: msstore apps are available from the Microsoft Store source from the Winget Package Manager
Winget packages (without specified source) have failed to detect their sources, anf therefore they are identified as winget.


Scoop does (if you are not using the nonportable bucket) install apps as portable.
Chocolatey may or may not, it depends on the package

**Reply by ppvnf** [reply]
I think there is a bug with .net packages (maybe because they have 86 and 64 bit versions )

<img width="956" alt="Screenshot 2023-10-12 215740" src="https://github.com/marticliment/WingetUI/assets/73800734/49b1fc95-27a7-4f64-8b51-d404ef9d313e">

I have opened https://github.com/marticliment/WingetUI/issues/1492, but I think this is slightly different

**Comment by elliot-100** [comment]
[Deleted, as I can't reproduce my problem (which was about packages other than dotnet) consistently]
